code/modules/admin/verbs/randomverbs.dm

```diff
@@ -516,7 +516,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 		else
 			M.add_ion_law(input)
 			for(var/mob/living/silicon/ai/O in mob_list)
-				O << "\red " + input + "\red...LAWS UPDATED"
+				O << "<font color='red'> </font>" + input + "<font color='red'>...LAWS UPDATED</font>"

```

Comment:
This is not correct. The goal with this macro was to make `input` red.

The correct line will look like this:

`O << "<font color='red'>" + input + " ...LAWS UPDATED</font>"`

Or alternatively, what I think looks better... although this doesn't make `input` red...

`O << input + "<font color='red'>... LAWS UPDATED!</font>"`